python/pipeline/stack.py

```diff
@@ -977,6 +978,129 @@ def make(self, key):
         self.insert1({**key, 'resized': resized, 'lcned': lcned, 'sharpened': sharpened})
 
 
+@schema
+class StackSurfaceMethod(dj.Lookup):
+    definition = """ # Methods used to compute surface of the brain
+
+    surface_method_id   : tinyint unsigned   # Unique ID given to each surface calculation method
+    ---
+    method_title        : varchar(32)        # Title of surface calculation method
+    method_description  : varchar(256)       # Details on surface calculation
+    """
+
+    contents = [
+        [1, 'Paraboloid Fit', 'Fit ax^2 + by^2 + cx + dy + f to surface after finding max of sobel']
+    ]
+
+
+@schema
+class StackSurface(dj.Computed):
+    definition = """ # Calculated surface of the brain
+
+    -> PreprocessedStack
+    -> StackSurfaceMethod
+    ---
+    surface_im        : longblob          # Matrix of calculated depth for each pixel in stack
```

Comment:
We don't have a policy of how big something has to be before we put em in an external. This is gonna be a h x w array, right? that's probably ok as a longblob but if you feel like it you can ask in the #datajoint slack channel what they recommend the maximum size per blob (or per row) be